### ISSUE TITLE
Error: A different version was loaded previously. You need to log out for changes to take effect

### ISSUE THREAD

**Issue by fcole90** [OttoAllmendinger/gnome-shell-screenshot#134]
On Gnome 3.36 (Ubuntu 20.04), updating the extension resulted in this error. I did not attempt to log-out and log-in again yet, as I have work to do, I'll update to say if it really fixes the error.
```
JS ERROR: Extension gnome-shell-screenshot@ttll.de: Error: A different version was loaded previously. You need to log out for changes to take effect.
loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:341:52
callback@resource:///org/gnome/shell/ui/extensionDownloader.js:234:39
gotExtensionZipFile/<@resource:///org/gnome/shell/ui/extensionDownloader.js:115:13
```

However, I think it would be better for the extension to fail in a more informative way. I had no idea why any update/install resulted in an error until checking the logs

**Comment by OttoAllmendinger** [comment]
Thanks for your report.

Did you update the extension using the browser plugin?

Are there any other extensions that you can update which do not result in the error state after update?

As far as I can tell, this was introduced in a recent change of `gjs`. I don't think I have any control over this behavior.

**Comment by fcole90** [comment]
Thanks for your extension and for taking care of it :blush:  I updated it through the plugin, yes, on extensions.gnome.org. I did not see any other extension with a similar message, but I don't have many from extensions.gnome.org. Most of the extensions I'm using are old-stable versions that Ubuntu syncs from Debian :relaxed: 

I'm sorry you might have no control over it. If this is the case, I might send a bug to gnome's maintainers of `gjs`
